go.mod

```
@@ -1,6 +1,6 @@
 module github.com/cloudoperators/concourse-oci-helm-chart-resource
 
-go 1.22.2
+go 1.23.4
```

```suggestion
go 1.23.0
```

we are using https://github.com/ldez/gomoddirectives in golangci-lint because anything else just yields a poor user experience when they haven't updated yet as eg gopls just refuses to work.